linera-chain/src/data_types.rs

```diff
@@ -675,6 +670,10 @@ impl Event {
 }
 
 impl ExecutedBlock {
+    pub fn messages(&self) -> &Vec<OutgoingMessage> {
```

Comment:
Minor: using the opportunity to add documentation :sweat_smile: 

```suggestion
    /// Returns the messages sent from this block.
    pub fn messages(&self) -> &Vec<OutgoingMessage> {
```